### ISSUE TITLE
No X videos can be downloaded

### ISSUE THREAD

**Issue by serial02** [Unrud/video-downloader#273]
Hello

 No videos from X (formerly Twitter) can be downloaded by Fedora 40 using the video-downloader application

![Captura de tela de 2024-05-18 13-25-39](https://github.com/Unrud/video-downloader/assets/32621790/8fb73e1c-18d2-4740-904f-f28b001511fb)

**Comment by yanlucas669** [comment]
my imaturity aside i did sum testin and discover a workaround
if you swamp the link to twitter dot com you can download your xvideos 
![image](https://github.com/Unrud/video-downloader/assets/85195780/06173770-141d-4bc5-97a3-3a10f3f7244c)

**Comment by Belarrius1** [comment]
Yeah, replace "x" by "twitter" in URL.

Can we get any update about this?

**Comment by mak448a** [comment]
It seems to be working right now for me. Does it work for you now?

**Comment by Belarrius1** [comment]
@mak448a Yes, it's works now, I using 0.12.21 and it's work with "x.com" you can close the issue, thanks!
